tests/fixtures/_utils.js

```diff
@@ -1,217 +1,336 @@
+        repeatable: true,
+      },
+    ],
+  };
+};
+
+const values = ctype => {
+  const results = {};
+  results[`${slugify(ctype)}-name`] = {
+    text: {
+      value: ipsum({
+        count: 1,
+        units: 'words',
+        format: 'plain',
+      }),
+    },
+  };
+  results[`${slugify(ctype)}-textblock`] = {
+    textblock: {
+      value: ipsum({
+        count: 2,
+        units: 'paragraphs',
+        format: 'plain',
+        sentenceUpperBound: 5,
+        paragraphUpperBound: 3,
+      }),
+    },
+  };
+  results[`${slugify(ctype)}-referencer`] = {
+    referencer: {
+      value: 'make-me-an-id',
+    },
+  };
+  results[`${slugify(ctype)}-referencer-dual`] = {
+    referencerdual1: {
+      value: 'make-me-an-id',
+    },
+    referencerdual2: {
+      value: 'make-me-an-id',
+    },
+  };
+  results[`${slugify(ctype)}-referencer-repeating`] = [
+    {
+      referencerrepeat: {
+        value: 'make-me-an-id',
+      },
+    },
+    {
+      referencerrepeat: {
+        value: 'make-me-an-id',
+      },
+    },
+  ];
+  results[`${slugify(ctype)}-referencer-dual-repeating`] = [
+    {
+      referencerdualrepeat1: {
+        value: 'make-me-an-id',
+      },
+      referencerdualrepeat2: {
+        value: 'make-me-an-id',
+      },
+    },
+    {
+      referencerdualrepeat1: {
+        value: 'make-me-an-id',
+      },
+      referencerdualrepeat2: {
+        value: 'make-me-an-id',
+      },
+    },
+    {
+      referencerdualrepeat1: {
+        value: 'make-me-an-id',
+      },
+      referencerdualrepeat2: {
+        value: 'make-me-an-id',
+      },
+    },
+  ];
+
+  return results;
+};
+
+/**
+ * Tests attributes with references
+ * @param  {object} t - ava testing
+ * @param  {object|array} attrs object containing attributes to test
+ */
+const referencer = (t, attrs) => {
+  if (typeof attrs !== 'object' || attrs === undefined) {
+    return;
   }
 
-  /**
-   * Add {type} fixtures to the content array
-   */
-  for (i = 0; i < t; i++) {
-    let referenced = '';
-    const name = ipsum({
-      count: 3,
-      units: 'words',
-      format: 'plain',
+  // an individual piece of content, so we need to test just the attributes
+  if (attrs.hasOwnProperty('attributes')) {
+    referencer(t, attrs.attributes);
+
+    return;
+  }
+
+  // an array of attributes, this set of attributes is a repeatable
+  if (Array.isArray(attrs)) {
+    attrs.forEach(attr => {
+      Object.keys(attr).forEach(atr => {
+        // an individual piece of content, so we need to test just the attributes
+        // at this point, we've found a piece of content _inside_ a piece of content's attributes
+        if (attr[atr].hasOwnProperty('attributes')) {
+          referencer(t, attr[atr].attributes);
+
+          return;
+        }
+
+        // we've reached our depth, test `meta` exists
+        else if (attr[atr].hasOwnProperty('id')) {
+          t.true(attr[atr].hasOwnProperty('meta'), 'attribute in array contains meta');
```

Comment:
Test that `meta` also has `url` inside it, and all of the goodies that go in to that